### PR TITLE
[FEATURE REQUEST] Replace "auto-uploads" with "automatic uploads"

### PR DESCRIPTION
**Pull request by jesmrec**
## Related Issues

App: https://github.com/owncloud/android/issues/4252

- [X] Add changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
- [x] Add feature to Release Notes in `ReleaseNotesViewModel.kt` creating a new `ReleaseNote()` with String resources (if required)

_____

### CODE REVIEW

**Comment by jesmrec** [comment]
Replace all "Camera uploads" with "Automatic uploads", not only in the UI but also in the code. 

Just one exception: names in file providers and database references were not change to prevent unnecessary migrations. Let me know if this is a good or a bad decision.

File: owncloudApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -58,9 +58,9 @@
     <string name="prefs_touches_with_other_visible_windows_summary">Allow touches when the view is obscured by another visible window. Enable it to use light filtering apps.</string>
     <string name="confirmation_touches_with_other_windows_title">Are you sure you want to enable this feature?</string>
     <string name="confirmation_touches_with_other_windows_message">Use this feature at your own risk. A malicious application could try to spoof you into unknowingly performing some actions, using other views.</string>
-    <string name="prefs_subsection_picture_uploads">Auto-upload pictures</string>
+    <string name="prefs_subsection_picture_uploads">Automatic picture uploads</string>
     <string name="prefs_subsection_picture_uploads_summary">Manage location and behavior of the pictures uploaded automatically</string>
-    <string name="prefs_subsection_video_uploads">Auto-upload videos</string>
+    <string name="prefs_subsection_video_uploads">Automatic video uploads</string>
```
Comment (JuancaG05) [inline owncloudApp/src/main/res/values/strings.xml:63]: I think you need to remove all the translations with the translations editor, othwerise Transifex won't detect that this is a new String and won't ask to translate it

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Just one detail @jesmrec 👍

File: owncloudApp/src/main/res/values/strings.xml (hunk 1 above)
Comment (jesmrec) [reply owncloudApp/src/main/res/values/strings.xml:63]: You're right, done!

**Review by JuancaG05 — APPROVED** [review]
LGTM!

**Comment by joragua** [comment]
Approved from my side. All works correctly 🚀

**Pull request merged**